### PR TITLE
[MM-40757] Release new Operator v1.17.0 in Cloud provisioner

### PR DESCRIPTION
**Pull request by stylianosrigas**
#### Summary
-  Release new Operator v1.17.0 in Cloud provisioner

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-40757

#### Release Note

```release-note
- Release new Operator v1.17.0 in Cloud provisioner
```

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1042,8 +1042,8 @@ github.com/mattermost/ldap v0.0.0-20191128190019-9f62ba4b8d4d/go.mod h1:HLbgMEI5
 github.com/mattermost/ldap v3.0.4+incompatible/go.mod h1:b4reDCcGpBxJ4WX0f224KFY+OR0npin7or7EFpeIko4=
 github.com/mattermost/mattermost-cloud v0.39.0/go.mod h1:GbWfZajyp+DvdEpoNfFeJ3ZplwTtDV9pl6PhNnp+wq4=
 github.com/mattermost/mattermost-operator v1.12.0/go.mod h1:a6pSJI6bDaIfO65M/Hvuqg8kUNYQQYDHLayoT5XZx5o=
-github.com/mattermost/mattermost-operator v1.16.0 h1:RZzuVdwVGBK3V2U3cBg4ZcvKiyU53gW8OynqYVDfDiU=
-github.com/mattermost/mattermost-operator v1.16.0/go.mod h1:pUS6pyAiW7511k9vH+9gLpo/KpHkDQfwRHTziVUmaCg=
+github.com/mattermost/mattermost-operator v1.17.0 h1:xd4wUUy1L3NqujcM4MLjhMpnNVfgewhq901uwOjbGxA=
```
Comment (spirosoik) [inline go.sum:1045]: are we sure that we need to update?

**Review by spirosoik — APPROVED** [review]
one comment other than that lgtm

**Review by Szymongib — APPROVED** [review]
Thanks for updating it!

File: go.sum (hunk 1 above)
Comment (stylianosrigas) [reply go.sum:1045]: Yeap :)

**Pull request merged**